GiEnJul/Controllers/GiverController.cs

```diff
@@ -0,0 +1,38 @@
+﻿using AutoMapper;
+using GiEnJul.Features;
+using Microsoft.AspNetCore.Mvc;
+using Serilog;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+// For more information on enabling Web API for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace GiEnJul.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class GiverController : ControllerBase
+    {
+        private readonly IGiverRepository _giverRepository;
+        private readonly ILogger _log;
+
+        public GiverController(IGiverRepository giverRepository, ILogger log)
+        {
+            _giverRepository = giverRepository;
+            _log = log;
+        }
+
+        
+        // POST api/<GiverController>
+        [HttpPost]
+        public async Task PostAsync([FromBody] Models.Giver giver)
+        {   
+            _log.Debug("Adding giver object: {@giver}", giver);
+            var result = await _giverRepository.InsertOrReplaceAsync(giver);
+            _log.Debug("Succesfully added giver: {@0}", result);
```

Comment:
I have two suggestions here:
Either remove unnecessary var
```suggestion
            _log.Debug("Adding giver object: {@giver}", giver);
            await _giverRepository.InsertOrReplaceAsync(giver);
            _log.Debug("Succesfully added giver: {@0}", result);
```
Or add return value. With inspiration from [documentation](https://docs.microsoft.com/en-us/aspnet/core/tutorials/first-web-api?view=aspnetcore-5.0&tabs=visual-studio#update-the-posttodoitem-create-method)
```suggestion
            //Change return type of function to Task<ActionResult<Giver>>
            _log.Debug("Adding giver object: {@giver}", giver);
            var result = await _giverRepository.InsertOrReplaceAsync(giver);
            _log.Debug("Succesfully added giver: {@0}", result);
            return CreatedAtAction(nameof(result), result);
```

Difference between the two is that frontend either receives an empty Http-Success response, or a Http-Success response with the added entity.

Reply:
Done! Changed to last suggestion